### PR TITLE
docs: Improve README Formatting

### PR DESCRIPTION
**Pull request by ueman**
Link to the rendered README https://github.com/ueman/home_widget/blob/7f255317b5f69bdd880708de12e0cdf32897e5a4/README.md

This uses the `<kbd>` HTML tag to render menu paths a bit nicer, see 
Screenshot 2024-01-10 at 07 32 58

I've also fixed and added syntax highlighting to a broken code fence.

### CODE REVIEW

**Comment by ueman** [comment]
@ABausG friendly ping to review these couple minor fixes

File: README.md [hunk 1]
```diff
@@ -67,6 +66,8 @@ let data = UserDefaults.init(suiteName:"YOUR_GROUP_ID")
 
 ### Create Widget Layout inside `android/app/src/main/res/layout`
 
+This file contains the 
+
```
Comment (ABausG) [inline README.md:70]: This seems like an unintentional change?
I'll be doing some adjustments to this section hopefully this week to finish up the work for documenting Jetpack Glance support so maybe remove this change?

**Review by ABausG — COMMENTED** [review]
Thanks for this! Love the `kbd` tags.

Just a small question regarding a line change you added where I believe something got mixed up

File: README.md (hunk 1 above)
Comment (ueman) [reply README.md:70]: Ah yes, this is a mistake 😅

```suggestion
```

**Review by ABausG — APPROVED** [review]

**Pull request merged**